src/bindings/python/tests/test_ngraph/test_variant.py

```diff
@@ -0,0 +1,86 @@
+from openvino.runtime import Variant
```

Comment:
we don't have Variant in new OV 2.0 API

See https://github.com/openvinotoolkit/openvino/pull/8520

Reply:
Here we have some misunderstanding. 
So I made another PR (#9084) where `ov::Any` is exposed as a `OVAny`.
In Python we can't  expose `ov::Any` as a `Any` because it is reserved by https://docs.python.org/3/library/typing.html#typing.Any